sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java

```diff
@@ -60,11 +53,7 @@ public void define(Context context) {
   }
 
   private RuleMetadataLoader getRuleMetadataLoader() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "getRuleMetadataLoader" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-254&issues=AWD-dbT4EFfjxkQcPwFu&open=AWD-dbT4EFfjxkQcPwFu)

[ISSUE_KEY]: AWD-dbT4EFfjxkQcPwFu